cmd/skopeo/login.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"io"
+	"os"
+
+	"github.com/containers/common/pkg/auth"
+	"github.com/containers/image/v5/types"
+	"github.com/spf13/cobra"
+)
+
+type loginOptions struct {
+	global    *globalOptions
+	loginOpts auth.LoginOptions
+	getLogin  optionalBool
+	tlsVerify optionalBool
+}
+
+func loginCmd(global *globalOptions) *cobra.Command {
+	opts := loginOptions{
+		global: global,
+		loginOpts: auth.LoginOptions{
+			Stdin:  os.Stdin,
+			Stdout: os.Stdout,
```

Comment:
This should only be set from the `run` method’s `stdout` parameter.

(Eventually the same for `Stdin`, but that can wait until we have tests that actually use it. But `Stdin` should at least be set in the same place, i.e. inside `run`.)